components/atom/switch/src/SwitchType/single.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import cx from 'classnames'
+import {BASE_CLASS, TYPES} from '../index'
+import AtomLabel from '@s-ui/react-atom-label'
+
+export const SingleSwitchTypeRender = (opts, isToggle, isFocus, isDisabled, focusSwitchCallback, blurSwitchCallback, toggleSwitchCallback) => {
```

Comment:
please treat this as a React component `({...your props...}) => (<div>....</div>)` and use it as one in your AtomSwitch.render function `<SingleSwitchTypeRender {...this.props} newProp1 newProp2... />`

Also define the propTypes

Same for the `ToggleSwitchTypeRender` fn